app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError, VaccineError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    success = 0
```

Comment:
You can remove the "success" counter because if one friend forgets their vaccine, the program will return "All friends must be vaccinated."(line 13)